test/parallel/test-cluster-shared-leak.js

```diff
@@ -15,14 +15,21 @@ if (cluster.isMaster) {
   worker1 = cluster.fork();
   worker1.on('message', common.mustCall(function() {
     worker2 = cluster.fork();
+    worker2.on('error', function(e) {
+      // EPIPE is OK on Windows
+      if (common.isWindows && e.code === 'EPIPE')
```

Comment:
I'm not too surprised it only happens on Windows but I don't understand why you only need to the check for the second worker.